src/reports/fetch-IR-strategy.ts

```diff
@@ -13,7 +13,7 @@ const streamPipeline = promisify(pipeline);
  * @param {*} rate
  * @param fileName the fileName to store the ir to
  */
-export async function fetchRateStrategyImage(rate) {
+export async function fetchRateStrategyImage(rate: any) {
```

Comment:
I think we would have a type for this as well here: https://github.com/bgd-labs/aave-cli/blob/main/src/reports/snapshot-types.ts#L66 would you mind adding?